code/src/App.js

```diff
@@ -47,8 +47,6 @@ export const App = () => {
       setMessageNew('')
     })
     .catch(error => console.log(error))
-    // .catch(() => setErrorMessage(prev => !prev))
-    //change this to error = console.error(error))?
```

Comment:
You can use any console outputs in production code. The console is only for debugging

Reply:
yes, accidentally pushed up comments I did not want to keep 👍